src/containers/AyatulKursi/index.js

```diff
@@ -50,9 +50,10 @@ const TopOptions = Loadable({
 });
 
 const title = 'Ayatul Kursi';
-const description = 'Ayatul Kursi is verse 255 of the second chapter, ' +
-                    'Surah al-Baqarah (The Chapter of the Cow), in the ' +
-                    'Holy Quran. It is also known as the Throne Verse.';
+const description =
```

Comment:
why not template literals?

Reply:
Sure,  next PR maybe. These changes are from lint auto fixer :)